src/main/kotlin/se/uulm/snowballr/backend/repository/association/ProjectMemberTableRepo.kt

```diff
@@ -76,6 +78,16 @@ interface IProjectMemberTableRepo {
      * @return A list of [ProjectMemberWithUser] objects, each containing a project member and the corresponding user information.
      */
     suspend fun getProjectMembersWithUsers(projectId: UUID): List<ProjectMemberWithUser>
+
+    /**
+     * Removes a project member from the project, according to the given project ID and user ID. This results in
+     * the deletion of the project member from the table.
+     *
+     * @param projectId The unique identifier of the project whose member should be removed.
+     * @param userId The unique identifier of the user, who should be removed.
+     * @return The number of deleted project members as [Int].
```

Comment:
```suggestion
     * Removes a project member with the given [userId] from the project with the given [projectId].
     *
     * @param projectId The unique identifier of the project where the member is part of.
     * @param userId The unique identifier of the user, who should be removed.
     * @return The number of deleted project members as [Int].
```
And another question: Is it really necessary to return the number of deleted project members?

Reply:
It is possible. I removed the return value.

Reply:
I think "Removes a project member with the given [userId] from the project with the given [projectId]." is enough as description. At the moment you state twice, that the entry is removed from the table which seems a bit unnecessary to me.

Reply:
I guess you are right. This is kind of "doppeltgemoppelt".